tools/hanabi_table.py

```diff
@@ -3,30 +3,37 @@
 from hanabi_hand import HanabiHand
 from hanabi_card import HanabiColor
 
+NUM_DISCLOSURES = 8
+NUM_MISTAKES = 3
+
 class HanabiTable:
-    def __init__(self, num_players, seed, is_rainbow_included):
+    RAINBOW_IS_WILD = 3
```

Comment:
`HanabiTable` needing to know this seems to indicate an enum would be nice here, though I can't remember ever using an enum in Python, so I don't know what the experience is like. Any reason to not use one in this case?

Reply:
my lame reason is I found enums to be dumb/too complex for what I needed. I was using enums for the color representation, which meant every reference to a color was making excessive calls.

Currently not inclined to fix this, but will make issue #15 to revisit